uniffi_bindgen/src/bindings/kotlin/gen_kotlin/miscellany.rs

```diff
@@ -5,54 +5,37 @@
 use crate::backend::{CodeOracle, CodeType, Literal};
 use askama::Template;
 use paste::paste;
-use std::fmt;
 
 #[allow(unused_imports)]
 use super::filters;
 
 macro_rules! impl_code_type_for_miscellany {
-     ($T:ty, $class_name:literal, $canonical_name:literal, $template_file:literal) => {
-         paste! {
-             #[derive(Template)]
-             #[template(syntax = "kt", escape = "none", path = $template_file )]
-             pub struct $T;
-
-             impl CodeType for $T  {
-                 fn type_label(&self, _oracle: &dyn CodeOracle) -> String {
-                     $class_name.into()
-                 }
-
-                 fn canonical_name(&self, _oracle: &dyn CodeOracle) -> String {
-                    $canonical_name.into()
+    ($T:ty, $class_name:literal, $canonical_name:literal, $template_file:literal) => {
```

Comment:
Do these macros disappear down in to one implementation now? (he asks, with some excitement)

Reply:
I think that might be possible as well.

Reply:
Do it!